### PR TITLE
Create HOW_TO_CREATE_A_PLAN

### PR DESCRIPTION
**Pull request by mallenexpensify**
### Details
Per this post in #contributor-management, we're going to be asking Contributors who post Feature Requests to use a simple template.  To provide more context, we want to point them (bookmark/pin in #expensify-open-source, likely link added to CONTRIBUTING.md) to a simplified version of our 'how to create a plan' in our manifesto. 

### Fixed Issues

$ GH_LINK

### Tests

1. Check to see if it's visible here - https://github.com/Expensify/App 

- [ ] Verify that no errors appear in the JS console

### QA Steps

- [ ] Verify that no errors appear in the JS console

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Android

cc @roryabraham @Beamanator who were helping

### CODE REVIEW

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:45]: I think we should also explain what is a Design doc or link to some explanation.

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:18]: I am just wondering, the direction we should avoid is Solution -> Problem, should the bad example be something like: 

Solution: I want a car.
Problem: We do not have a car.

But this is just coming to my mind given the order of the problem/solution issue. Let me know what you think.

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:25]: I think there are two spaces here. Suggestion
```suggestion
The broader and better contextualized the problem definition, the greater opportunity for discovering radical, out-of-the-box solutions. The odds of achieving something are dramatically improved by clearly defining what it is. Take at least as much care perfecting your problem definition as you do the solution, because the perfect solution to the wrong problem is typically an awful solution to the right one.
```

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:37]: Two spaces too.
```suggestion
If you are finding the solution to be difficult, go back and beat harder on the problem to break it up into smaller pieces. Keep repeating until you have a general list of prioritized stages, with early stages solving the dependencies required by later stages, each of which is extremely well defined, with a reasonably obvious preferred solution.
```

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:40]: Also more spaces.
```suggestion
Have a trusted peer or two proof your P/S statement and help you ensure it is well-defined. If you're in need of a peer to proof, post in #expensify-open-source to ask for help. Refine it and then share with another peer or two until you have a clear, understandable P/S statement. The more complex the problem and solution, the more people should review it. Keep going back to step 1 if needed.
```

**Review by mountiny — COMMENTED** [review]
@mallenexpensify No worries! I have fixed it! The file was correctly at the root dictionary, but you also need to include the `.md` suffix to indicate this is Markdown file.

No wit should be good and I have included some sugegstions. Nice guidelines!

File: HOW_TO_CREATE_A_PLAN.md
Comment (mallenexpensify) [reply HOW_TO_CREATE_A_PLAN.md:45]: Contributors won't have the context of a design doc so I don't want to go into too much detail yet.  What about updating to "an internal Design Doc may be required." to stress it's not something a contributor should be looking for?

> link to some explanation

I think if add 'internal' we don't need more detail. But.. I initially read it as 'link to some examples'.  Once we have a few good examples in #expensify-open-source we should add links to them.  Or... create a new .md file with a handful of examples (since not everyone will be in #expensify-open-source)

Comment (mountiny) [reply HOW_TO_CREATE_A_PLAN.md:45]: Agreed! You are right, it is internal, but it might be good to imply that they basically _do not have to know what it is_, kind of.

File: HOW_TO_CREATE_A_PLAN.md
Comment (mallenexpensify) [reply HOW_TO_CREATE_A_PLAN.md:18]: I think you're right cuz it's "reverse-engineer a Solution" which means you would start with the solution to get the problem.. right?  If so, can you make that update?

**Comment by mallenexpensify** [comment]
Thanks for the quick review and feedback/suggestions @mountiny

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:45]: @mallenexpensify Adding the suggestion in that case. 
```suggestion
Depending on the complexity of the P/S Statement, an internal Design Doc may be required.
```

**Review by mountiny — COMMENTED** [review]
One last change then and we can :shipit:  🎉

File: HOW_TO_CREATE_A_PLAN.md
Comment (Beamanator) [inline HOW_TO_CREATE_A_PLAN.md:9]:
```suggestion
It’s also important to be familiar with our existing documentation including [CONTRIBUTING.md](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md) and [README.md](https://github.com/Expensify/App/blob/main/README.md)
```

File: HOW_TO_CREATE_A_PLAN.md
Comment (Beamanator) [reply HOW_TO_CREATE_A_PLAN.md:18]: I like it 👍

File: HOW_TO_CREATE_A_PLAN.md
Comment (Beamanator) [inline HOW_TO_CREATE_A_PLAN.md:20]: I think this is slightly clearer

```suggestion
- Keep the language simple - Reduce every word down to its most plain and straightforward manner.  Avoid jargon. If you use abbreviations, include the full spelling the first time.
```

File: HOW_TO_CREATE_A_PLAN.md
Comment (Beamanator) [inline HOW_TO_CREATE_A_PLAN.md:31]: I guess we should define this since above we say define acronyms :D

```suggestion
- ROI (return on investment) - the current value of the problem, versus the cost of your solution
```

File: HOW_TO_CREATE_A_PLAN.md
Comment (Beamanator) [reply HOW_TO_CREATE_A_PLAN.md:45]: What if we add `(Expensify employees will take it from there)` to make it clear that at that point it's out of the contributor's hands?

**Review by Beamanator — COMMENTED** [review]
Lookin shnazzy! Just added a few little ideas

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [reply HOW_TO_CREATE_A_PLAN.md:45]: @Beamanator I have updated this suggestion, with your suggestion, to form our mutual suggestion! 😄

File: HOW_TO_CREATE_A_PLAN.md
Comment (mountiny) [inline HOW_TO_CREATE_A_PLAN.md:18]:
```suggestion
- Resist the temptation to reverse-engineer a Solution into a matching Problem.  (ie. Solution: Let's get a car! Problem: We don’t have a car.) 
```

**Review by mountiny — COMMENTED** [review]
Added that one suggestion I forgot to suggest.

**Review by mountiny — APPROVED** [review]
LGTM after discussing in NewDot, I will merge and not wait for other review!

**Comment by Beamanator** [comment]
Niiice way to get these changes in & merged y'all 💪 👍

**Pull request merged**
